frontend/kolide/endpoints.ts

```diff
@@ -44,4 +44,10 @@ export default {
   TEAMS_MEMBERS: (teamId: number) => {
     return `/v1/fleet/teams/${teamId}/users`;
   },
+  TEAMS_TRANSFER_HOSTS: (teamId: number) => {
+    return `/v1/fleet/teams/${teamId}/hosts`;
+  },
+  TEAMS_ENROL_SECRETS: (teamId: number) => {
```

Comment:
Prefer US English spelling of "enroll" with 2 `l`s.

Reply:
ah k. noticed my computer marks it as incorrect spelling when I use US spelling lol